### PR TITLE
implemented spring mvc configuration. Added UserController, UserResponseDto

### PR DESCRIPTION
**Pull request by andriylagun**
In method inject return type is RedirectView, for automatically redirection on userList page.

### CODE REVIEW

File: src/main/java/com/online/cinema/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                .build());
+        userService.add(User.builder().name("John")
+                .email("john@gmail.com")
+                .password("12345")
+                .build());
+        userService.add(User.builder().name("Anastasia")
+                .email("anastasia@gmail.com")
+                .password("12345")
+                .build());
+        return new RedirectView("/user");
```
Comment (tarkaiev) [inline src/main/java/com/online/cinema/controller/UserController.java:43]: good job with redirect

File: src/main/java/com/online/cinema/dao/UserDao.java [hunk 2]
```diff
@@ -6,5 +6,7 @@
 public interface UserDao {
     User add(User user);
 
+    User get(Long id);
```
Comment (tarkaiev) [inline src/main/java/com/online/cinema/dao/UserDao.java:9]: according to checklist we don't have to change dao and service layers

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<p>HELLO WORLD!</p>
+</body>
+</html>
```
Comment (tarkaiev) [inline src/main/webapp/WEB-INF/views/home.jsp]: do we need this page in this PR?

**Review by tarkaiev — APPROVED** [review]

File: src/main/java/com/online/cinema/config/WebConfig.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.online.cinema.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages =
+        "com.online.cinema.controller")
```
Comment (Sofasmile) [inline src/main/java/com/online/cinema/config/WebConfig.java]:
```suggestion
@ComponentScan(basePackages = "com.online.cinema.controller")
```

File: src/main/java/com/online/cinema/controller/UserController.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.view.RedirectView;
+
+@RequestMapping("/user")
+@RestController
+public class UserController {
+    private UserService userService;
+
+    @Autowired
+    public void setUserService(UserService userService) {
```
Comment (Sofasmile) [inline src/main/java/com/online/cinema/controller/UserController.java]: let's use constructor here

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<%--
```
Comment (Sofasmile) [inline src/main/webapp/WEB-INF/views/home.jsp]: let;'s remove jsp page

Comment (andriylagun) [reply src/main/webapp/WEB-INF/views/home.jsp]: done

File: src/main/java/com/online/cinema/controller/UserController.java (hunk 5 above)
Comment (andriylagun) [reply src/main/java/com/online/cinema/controller/UserController.java]: done

File: src/main/java/com/online/cinema/config/WebConfig.java (hunk 4 above)
Comment (andriylagun) [reply src/main/java/com/online/cinema/config/WebConfig.java]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**